src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolver/DateTimeValueResolverTest.php

```diff
@@ -149,6 +171,16 @@ public function test404Exception(ArgumentMetadata $argument, Request $request)
         iterator_to_array($results);
     }
 
+    protected function setUp(): void
```

Comment:
Should be the first methods in the class to be consistent with our CS.

Reply:
Fixed.